CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to the "vscode-servicefabric" extension will be documented i
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## Version [0.1.6]
+- Fix security vulnerability in `diff` package
```

Comment:
Should rather be: Updating diff package to "0.1.6" to which has a security fix.

Reply:
Thanks, edited.